.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug Report
+about: Template for reporting bugs
+title: ''
+labels: bug
+assignees: ''
+
+---
```

Comment:
I think we are missing some rules section, that could be added to all issue templates. Could be something like:

-------------------

**WARNING: Please, read this carefully before submitting an issue**
It is important to realize that this is NOT A SUPPORT FORUM, this is for ISSUES ONLY. If you need support, have questions you should read the [documentation](https://docs.cartesi.io/) or ask in `#cartesi-machine` in [Cartesi Discord server](https://discord.gg/cartesi).

Please, before submitting a new issue, verify the following:

- You have tested the issue on latest release, usually the master branch.
- You have verified that there is no similar issue already reported.
- You have verified, with your best effort, that your code or use case doesn't contain errors or misuse.